sample-customFunction-remote/src/main/resources/jettyMavenTest.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0"?>
```

Comment:
@beebs-systap @igor-kim @krokhaleva Do we need this if we are using web-override.xml?